src/main/java/mate/controller/car/AddDriverToCarController.java

```diff
@@ -11,6 +12,7 @@
 import mate.service.CarService;
 import mate.service.DriverService;
 
+@WebServlet("/cars/drivers/add")
 public class AddDriverToCarController extends HttpServlet {
     private static final Injector injector = Injector.getInstance("mate");
     private final CarService carService = (CarService) injector
```

Comment:
line 37 - don't you think it's better to send to all the cars  with drivers pages as after added a driver the user sees the same page ?